debian/rules

```diff
@@ -42,7 +42,6 @@ override_dh_install:
 	find debian/tmp -name \*.la -delete
 	find debian/tmp -name libdummy-indicator\* -delete
 	rm -f debian/tmp/gtk2/usr/share/libayatana-indicator/80indicator-debugging
-	rm -f debian/tmp/gtk2/usr/lib/systemd/user/ayatana-indicators-pre.target
```

Comment:
Now I know where the code in https://github.com/AyatanaIndicators/libayatana-common/pull/26/commits/b65e02ae2aa4aa1693808fa9c4ba3f64bf594aec (PR libayatana-common#26) comes from.

I think that the above "rm -f" is an old and rotten remnant that noone ever noticed as no-op code for a while.

The original reason for it was: both GTK-{2,3} builds produced an ayatana-indicators-pre.target file, only the GTK-3 one got used. So, the GTK-2 one needed removing because of dh_missing.

Then ayatana-indicators-pre.target got removed to ayatana-indicators.target. And the above line got forgotten until today.

It is safe to remove this line and to not port it over to libayatana-common#26, simply drop that override_dh_install bit there. Thanks.